packages/flutter_tools/lib/src/macos/cocoapod_utils.dart

```diff
@@ -31,6 +31,12 @@ Future<void> processPodsIfNeeded(
     iosPlatform: project.ios.existsSync(),
     macOSPlatform: project.macos.existsSync(),
     forceCocoaPodsOnly: forceCocoaPodsOnly,
+    // TODO(matanlurey): As-per discussion on https://github.com/flutter/flutter/pull/157393
+    // we'll assume that iOS/MacOS builds do not use or rely on the `.flutter-plugins` legacy
+    // file being generated. A better long-term fix would be not to have a call to refreshPluginsList
+    // at all, and instead have it implicitly run by the FlutterCommand instead. See
+    // https://github.com/flutter/flutter/issues/157391 for details.
```

Comment:
Turbo-super-nit:

VSCode will parse all of this as a single todo if you add a space at the start of the proceeding lines:

```suggestion
    // TODO(matanlurey): As-per discussion on https://github.com/flutter/flutter/pull/157393
    //  we'll assume that iOS/MacOS builds do not use or rely on the `.flutter-plugins` legacy
    //  file being generated. A better long-term fix would be not to have a call to refreshPluginsList
    //  at all, and instead have it implicitly run by the FlutterCommand instead. See
    //  https://github.com/flutter/flutter/issues/157391 for details.
```

Then again, I don't know if I actually want more squiggly lines in my editor 🤷